### ISSUE TITLE
Unable to download/stream programs

### ISSUE THREAD

**Issue by folence** [z411/onsen-watch#1]
![onsen-watch_2019-05-28_10-34-07](https://user-images.githubusercontent.com/11154411/58463812-d39bdb00-8134-11e9-9f67-d6b8052cce8d.png)
There is probably just something that has broken the functions of this program in the last three years of no further development. If you could have a look to see if maybe its not too big a problem.
Figured id hear if you were interested in fixing this, as it looks like a great program, rather than starting my own project and sinking a lot of time on something inferior.

**Comment by z411** [comment]
Hi, thanks for the heads up. It must be some recent change because I remember using it recently. I'll definitely look into it as soon as I can, thanks.

**Comment by z411** [comment]
It seems to be working fine for me:
![onsen-watch_2019-05-29_07-27-39](https://user-images.githubusercontent.com/1988512/58553669-89256780-81e3-11e9-94e6-7cafe655a63c.png)

Can you access the following link?
https://onsen-dl.sslcs.cdngc.net/radio/otomain190523wV5i.mp3

**Comment by folence** [comment]
Yup, can access the link without any problem. Using the onsen site in general is no problem (except having to use chrome instead of ff and enabling flash)
